src/mapper/map.rs

```diff
@@ -323,26 +361,6 @@ impl EventWriter {
             };
         }
 
-        // TODO: add witness set data to transaction
```

Comment:
Note that witnesses are actually attached to blocks in Pallas and not to transactions. 
I don't know if this could be an issue later or not.

Reply:
Right, Pallas attaches witnesses to blocks because it follows the layout of the CBOR serialization structure. From the Oura event model perspective, I think it makes more sense to attach it to the corresponding transaction.